README.md

```diff
@@ -3,4 +3,10 @@
 
 ## Wstęp
 
-Strona jest stworzona z wykorzystaniem technologii [HUGO](https://gohugo.io/about/).
\ No newline at end of file
+Strona jest stworzona z wykorzystaniem technologii [HUGO](https://gohugo.io/about/).
+
+## Instalacja
+
+1. Zainstaluj [Hugo - extended edition](https://gohugo.io/installation/)
+2. Sklonuj to repozytorium
+3. Aby uruchomić serwer deweloperski: `hugo server`
```

Comment:
**thought**: With set  `baseurl` in [config](https://github.com/kozako2/gildia-organizatorow-website/blob/main/config.toml#L1) i thought that using `-b` is required.

```
3. Aby uruchomić serwer deweloperski: `hugo server -b "/"`
```
However after local check, this is not necessary.